poppler-sys/build.rs

```diff
@@ -314,7 +317,13 @@ const WHITELIST_POPPLER: [&str; 73] = [
     "_PopplerTextSpan",
     "PopplerTextSpan",
 ];
-const WHITELIST_POPPLER_DOCUMENT: [&str; 2] = ["goffset", "gint64"];
+const WHITELIST_POPPLER_DOCUMENT: [&str; 4] = [
+    "goffset",
+    "gint64",
+    //
+    "_PopplerPageRange",
+    "PopplerPageRange",
```

Comment:
Whitelisting those types here prevents the opaque version of the type from being re-defined for each bindgen pass (for each module) - this is because each whitelist becomes a blacklist for each other module.